seqr/utils/elasticsearch/es_utils_2_3_tests.py

```diff
@@ -1333,7 +1094,9 @@ def test_invalid_get_es_variants(self):
         self.mock_es_client.indices.get_mapping.side_effect = lambda **kwargs: {}
         with self.assertRaises(InvalidIndexException) as cm:
             get_es_variants(results_model)
-        self.assertEqual(str(cm.exception), 'Could not find expected indices: test_index_sv, test_index')
+        message = str(cm.exception).split(': ')
+        self.assertEqual(message[0], 'Could not find expected indices')
+        self.assertSetEqual(set(message[1].split(', ')), {'test_index_sv', 'test_index'})
```

Comment:
I thought you sorted the indices so this would be a predictable order

Reply:
I removed the sorting in es_search.py which looks like below:
```
            raise InvalidIndexException('Could not find expected indices: {}'.format(
                ', '.join(set(self._indices) - set(self.index_metadata.keys()))
            ))
```
Although `self._indices` is sorted, the element order of `set(self._indices) - set(self.index_metadata.keys())` can change each run.

Reply:
then you should sort `set(self._indices) - set(self.index_metadata.keys())`

Reply:
Okay, I will change them back.